modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/benchmarks/TpchHelper.java

```diff
@@ -65,8 +65,16 @@ public static String getQuery(String queryId) {
         }
     }
 
+    /**
+     * Returns a string representing DDL script with all the tables' and indexes' definitions
```

Comment:
what does mean symbols ' in the sentence?

Reply:
"An apostrophe followed by an “s” is the most common way to indicate possession (ownership) in English"